package.json

```diff
@@ -98,9 +98,9 @@
     "request-as-curl": "0.1.0",
     "request-promise": "4.2.3",
     "sanitize-html": "1.20.1",
-    "sequelize": "4.43.0",
-    "sequelize-cli": "4.0.0",
-    "sequelize-temporal": "1.0.6",
+    "sequelize": "^5.7.6",
```

Comment:
We use fixed versions. Should be `5.7.6` or whatever is the most recent.

Reply:
Done